### PR TITLE
become an aditor removed from Navbar

### PR DESCRIPTION
**Pull request by ferhataltinisik**
Become an aditor removed from Navbar

### CODE REVIEW

**Review by milleymilli — COMMENTED** [review]
backend/Dockerfile   ,,,,,, reason for editing?

**Review by milleymilli — COMMENTED** [review]
i have reviewed it.

**Comment by ferhataltinisik** [comment]
> backend/Dockerfile ,,,,,, reason for editing?

I changed it and updated

File: frontend/src/Components/TopNav/index.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ class TopNav extends Component {
             </Grid>
             {!this.props.user.success && (
               <p className="Editor-text">
-                Become an editor{' '}
+                {/*Become an editor{' '}*/}
```
Comment (jimijos) [inline frontend/src/Components/TopNav/index.js:109]: Perfect

**Review by jimijos — APPROVED** [review]

**Review by milleymilli — APPROVED** [review]

**Comment by ferhataltinisik** [comment]
cancelled

**Pull request closed without merging**